test/api/unit/services/Mailer.test.js

```diff
@@ -113,20 +122,37 @@ describe('mailer', () => {
         const sandboxNextCleaningAt = moment().add(expiryDays, 'days');
         const dateStr = sandboxNextCleaningAt.format('MMMM DD, YYYY');
         const org = await createSandboxOrg(sandboxNextCleaningAt.toDate());
-        const newUser = await factory.user();
+        const newUser = await createUserWithUAAIdentity();
         await org.addUser(newUser, { through: { roleId: userRole.id } });
         Mailer.init();
+
         const jobs = await Mailer.sendSandboxReminder(org);
         jobs.forEach((job) => {
           expect(job.name).to.eq('sandbox-reminder');
-          expect(org.Users.find(u => job.data.to.includes(u.email))).to.not.be.null;
+          expect(org.Users.find(u => job.data.to.includes(u.UAAIdentity.email))).to.not.be.null;
           expect(job.data.subject).to.eq(`Your Pages sandbox organization\'s sites will be removed in ${expiryDays} days`);
           expect(job.data.html).to.eq(Templates.sandboxReminder({
             organizationId: org.id, dateStr, organizationName: org.name, hostname, sites: org.Sites,
           }));
         });
         expect(jobs.length).to.equal(org.Users.length);
       });
+
+      it('`sandbox-reminder` throws an error if a user lacks UAAIdentity', async () => {
+        const expiryDays = 5;
+        const sandboxNextCleaningAt = moment().add(expiryDays, 'days');
+        const org = await createSandboxOrg(sandboxNextCleaningAt.toDate());
+        const nonUAAUser = await factory.user();
+        await org.addUser(nonUAAUser, { through: { roleId: userRole.id } });
+        await org.reload();
+        Mailer.init();
+
+        const error = await Mailer.sendSandboxReminder(org).catch(e => e);
+
+        expect(error).to.be.an('error');
+        expect(error.message).to.contain(`Failed to queue a sandbox reminders for organization@id=${org.id}`);
```

Comment:
```suggestion
        expect(error.message).to.contain(`Failed to queue a sandbox reminder for organization@id=${org.id}`);
```

Reply:
Ha, that's a consequence of the error aggregation that was already in there. It can be inflected.

Reply:
or you could replace `a` with `one or more` reminders?

Reply:
Is the approach I took OK with you?

Reply:
Specifically: 

```js
const errMsg = [
  `Failed to queue ${errors.length === 1 ? 'a sandbox reminder' : 'sandbox reminders'} `
  + `for organization@id=${organizationId} members:\n`,
  errors.map(e => `  user@id=${e.item.id}: ${e.message}`).join('\n'),
].join();
```

and

```js
expect(error.message).to.contain(`Failed to queue a sandbox reminder for organization@id=${org.id}`);
```